lib/material/material_gui.flow

```diff
@@ -454,14 +461,103 @@ MAttachHeight2(m : Material, height : DynamicBehaviour<double>, c : Material) ->
 }
 
 MAttachToFirst(first : Material, second : Material) -> Material {
-	wh = make(WidthHeight(0.0, 0.0));
+	wh = makeWH();
 	MGroup2(
 		MAttachBox(first, wh),
-		MAvailable(
-			MFixSize(second, MFixed(0.0, 0.0)),
-			TSized(wh)
-		)
+		MAvailable(second, TSized(wh))
```

Comment:
If the second is bigger than the first, and might be growing and changing over time, will that not cause the available for the first to change, against the desire? The idea is that the first one will NOT be affected by the second in any case.